ext/drbbdup/drbbdup.h

```diff
@@ -264,6 +264,11 @@ typedef struct {
      * in order to direct control to the appropriate basic block.
      */
     opnd_t runtime_case_opnd;
+    /**
+     * Instructs drbbdup whether or not the loading of the runtime case should be
+     * locked/atomic.
+     */
+    bool do_lock;
```

Comment:
(Made it to computer so repeating in proper location:) Suggesting a name like "atomic_load_encoding".

Reply:
Done.